EIPS/eip-2579.md

```diff
@@ -0,0 +1,128 @@
+---
+eip: 2579
+title: Merkle Tree Generalized Index
+author: naxe (ngmachado @LogicB0x)
+discussions-to: https://ethereum-magicians.org/t/merkle-tree-generalized-index/4162/
+status: Draft
+type: Standards Track
+category: ERC
+created: 2020-03-31
+---
+
+## Simple Summary
+
+Using a Merkle Tree with a Generalized Index coordination system as used in other systems with the necessary adaptation to a smart contract environment, reducing the computation requirement and storage space.
+
+## Motivation
```

Comment:
```suggestion
## Abstract
...

## Motivation
```
An abstract would be quite valuable for this EIP.  Without digging into the specification, it is not clear to me as a reader exactly what this EIP does.